src/app/material-timepicker/services/time-adapter.ts

```diff
@@ -25,7 +25,9 @@ export class TimeAdapter {
         }
         const {format} = opts;
         const parsedTime = TimeAdapter.parseTime(time, opts).setLocale(TimeAdapter.DEFAULT_LOCALE);
-
+        if (parsedTime.invalid) {
+            return null;
```

Comment:
I'd like to point out that I prefer `null` here over `'Invalid Time'`, otherwise the invalid value entered by the user will be replaced with 'Invalid Time' in the input field, which is not nice as the user won't be able to correct his/her mistake. I'd leave it to the developer to put form validation on the input field.